README.md

```diff
@@ -37,9 +37,11 @@ steps:
         https://myrepo.com/conan-config.git
 
   - name: Install Conan dependencies
-    run: conan install . --build=missing
+    run: conan install . --lockfile=conan.lock --lockfile-partial --lockfile-out=conan.lock --build=missing
```

Comment:
Depending on the expectations, if the user is able to change the dependencies versions at anytime and things are expected to keep working, then good. If we want the lockfile to actually fail if user changes dependencies versions (there are organizations that might want that to avoid developers to do that), then better only ``--lockfile=conan.lock``. 

I think it would use that later in the code, and then explain the ``conan install . --lockfile=conan.lock --lockfile-partial --lockfile-out=conan.lock`` use case in text.

Reply:
Understood, I'll add a small snippet below with the lockfile example.

Reply:
@memsharded I just added a separate section for lockfile now in the commit [9273689](https://github.com/conan-io/setup-conan/pull/3/commits/92736898141df3dd695ad8e763304768d01b016b)